clouddriver-ecs/src/integration/java/com/netflix/spinnaker/clouddriver/ecs/test/CreateServerGroupSpec.java

```diff
@@ -86,6 +100,59 @@ public void setup() {
                   return new DescribeTargetGroupsResult().withTargetGroups(testTg);
                 });
 
+    when(mockAWSApplicationAutoScalingClient.describeScalableTargets(
```

Comment:
since this test case requires different mocked responses & setup, I'd prefer if we split it off into it's own file so that the other test cases continue to operate with minimally-mocked data. (Also I anticipate we'll have more than one test case w/ app autoscaling in the future)